packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -327,6 +372,121 @@ const dummyEntityTypeInitializer = async (graphApi: GraphApi) => {
   })(graphApi);
 };
 
+const archivedPropertyTypeInitializer = propertyTypeInitializer({
+  namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+  title: "Archived",
+  description: "Whether or not something has been archived.",
+  possibleValues: [{ primitiveDataType: "Boolean" }],
+});
+
+const summaryPropertyTypeInitializer = propertyTypeInitializer({
+  namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+  title: "Summary",
+  description: "The summary of the something.",
+  possibleValues: [{ primitiveDataType: "Text" }],
+});
+
+const titlePropertyTypeInitializer = propertyTypeInitializer({
+  namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+  title: "Summary",
+  description: "The title of something.",
+  possibleValues: [{ primitiveDataType: "Text" }],
+});
+
+const indexPropertyTypeInitializer = propertyTypeInitializer({
```

Comment:
I suppose this can go as well

Reply:
https://github.com/hashintel/hash/pull/1081#discussion_r979855048